R/sof.mpm2.R

```diff
@@ -60,25 +61,31 @@ makeMPM2Function = function(n.peaks, dimensions, topology, seed, rotated = TRUE,
   # build parameter set (bounds are [0, 1]^d)
   par.set = makeNumericParamSet("x", len = dimensions, lower = 0, upper = 1)
 
-  # import rPython namespace
-  BBmisc::requirePackages("_rPython", why = "smoof::makeMultiplePeaksModel2Function")
+  # import reticulate namespace
+  BBmisc::requirePackages("_reticulate", why = "smoof::makeMultiplePeaksModel2Function")
+
+  # FIXME: initialize 3 functions from mpm2.py as NULL such that they have a visible binding when checking the pkg
```

Comment:
What is this FIXME about?
As far as I understand the following lines actually achieve what is described here. Hence, it is no FIXME anymore? :-)

Reply:
good point, probably this was still there to check whether this addition was actually needed..